tests/rogo_test/principalparts_question_test.py

```
@@ -124,7 +135,7 @@ def test_principalparts_question():
         assert type(output) is PrincipalPartsQuestion
 
         assert output.check(output.principal_parts)
-        ic(output)  # type: ignore[name-defined] # noqa: F821
+        ic(output)  # noqa: F821
```

Pylint (3.3.7)

[error] Undefined variable 'ic'

(E0602)